sdk/cognitivelanguage/ai-language-text/test/public/utils/customTestHelpter.ts

```diff
@@ -0,0 +1,181 @@
+  /* Return a container client */
+  const defaultAzureCredential = new DefaultAzureCredential();
+  // Get documents containers
+  const blobServiceClient = new BlobServiceClient(
+    process.env["STORAGE_ENDPOINT"] || "string",
+    defaultAzureCredential
+  );
+  return blobServiceClient.getContainerClient(storageInputContainerName);
+}
+
+async function uploadDocuments(fileName: string): Promise<void> {
+  const client = await createStorageBlob();
+
+  // Unpack zip files and upload to blob
+  await decompress(path.join(__dirname, `${fileName}.zip`), __dirname);
+  const dir = path.join(__dirname, fileName);
+  fs.readdir(dir, function (err, files) {
+    if (err) {
+      console.log("Error while listing", err);
+      return;
+    }
+    files.forEach(async function (fileName) {
+      if (fileName.endsWith(".txt")) {
+        let blob = client.getBlockBlobClient(fileName);
+        let filePath = path.join(dir, fileName);
+        await blob.uploadFile(filePath);
+      }
+    });
+  });
+}
+
+async function polling(client: TextAuthoringClient, project: any) {
+  try {
+    const poller = getLongRunningPoller(client, project);
+    await poller.pollUntilDone();
+  } catch (error) {
+    console.log("Poller encountered error", error);
+  }
+}
+
+export async function createCustomTestProject(
+  client: TextAuthoringClient,
+  projectKind:
+    | "CustomEntityRecognition"
+    | "CustomSingleLabelClassification"
+    | "CustomMultiLabelClassification"
+    | "CustomAbstractiveSummarization",
+  projectName: string,
+  deploymentName: string
+) {
+  let files: string;
+  let assets:
+    | ExportedCustomSingleLabelClassificationProjectAssets
+    | ExportedCustomMultiLabelClassificationProjectAssets
+    | ExportedCustomEntityRecognitionProjectAssets;
+  switch (projectKind) {
+    case "CustomSingleLabelClassification":
+      assets = customSingleLabelAssets;
+      files = "WebOfScience";
+      break;
+
+    case "CustomMultiLabelClassification":
+      assets = customMultiLabelAssets;
+      files = "";
+      break;
+
+    case "CustomEntityRecognition":
+      assets = customEntityAssets;
+      files = "LoanAgreements";
+      break;
+
+    default:
+      assets = { projectKind: "CustomSingleLabelClassification" };
+      files = "";
+      break;
+  }
+  const modelLabel = "projectModel";
+  await uploadDocuments(files);
+
+  const options: CreateProjectOptions = {
+    projectKind,
+    storageInputContainerName,
+    projectName,
+    language,
+    multilingual: false,
+    settings: {},
+  };
+
+  // Create a project
+  const importTask = await client
+    .path("/authoring/analyze-text/projects/{projectName}/:import", projectName)
+    .post({
+      body: {
+        metadata: options,
+        projectFileVersion: "2022-05-01",
+        stringIndexType: "Utf16CodeUnit",
+        assets,
+      },
+    });
+  await polling(client, importTask);
+  console.log("Project created");
```

Comment:
No need to print to console but feel free to log using the Azure logger if you feel like it is needed.
```suggestion
```